tests/integration/test_logcollector/test_location/test_exclude.py

```diff
@@ -0,0 +1,255 @@
+                   '/tmp/wazuh-testing/test2.txt'],
+         'log_format': 'syslog', 'exclude': '/tmp/wazuh-testing/*test*',
+         'expected_matches': ['none'],
+         'description': 'Testing right and left wildcard, right and left exclude'},
+        {'location': '/tmp/wazuh-testing/*',
+         'files': ['/tmp/wazuh-testing/1test1.txt', '/tmp/wazuh-testing/1test1.log', '/tmp/wazuh-testing/2test2.log',
+                   '/tmp/wazuh-testing/test2.txt'],
+         'log_format': 'syslog', 'exclude': '/tmp/wazuh-testing/*',
+         'expected_matches': ['none'],
+         'description': 'Testing wildcard location'}
+
+    ]
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f"{x['LOCATION'], x['LOG_FORMAT']}" for x in parameters]
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+
+# Fixtures
+@pytest.fixture(scope="module")
+def create_directory():
+    """Create expected directories."""
+    if sys.platform == 'win32':
+        os.makedirs(fr'{temp_dir}\wazuh-testing', exist_ok=True)
+    else:
+        os.makedirs('/tmp/wazuh-testing', exist_ok=True)
+    yield
+
+    if sys.platform == 'win32':
+        rmtree(fr'{temp_dir}\wazuh-testing', ignore_errors=True)
+    else:
+        rmtree('/tmp/wazuh-testing')
+
+
+@pytest.fixture(scope='module', params=configurations, ids=configuration_ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope="module")
+def get_local_internal_options():
+    """Get configurations from the module."""
+    return local_internal_options
+
+
+@pytest.fixture(scope='module')
+def create_files(request, get_configuration):
+    """Create expected files."""
+    files = get_configuration['metadata']['files']
+
+    for file_location in files:
+        with open(file_location, 'w+') as file:
+            file.write(' ')
+    yield
+
+    for file_location in files:
+        if os.path.exists(file_location):
+            os.remove(file_location)
+
+
+def test_exclude(get_local_internal_options, configure_local_internal_options, create_directory, create_files,
+                 get_configuration, configure_environment,
+                 restart_logcollector):
+    """Check if logcollector is excluding specified files.
+
+    Raises:
+        TimeoutError: If the expected callback is not generated.
+    """
+    files = get_configuration['metadata']['files']
+    excluded = get_configuration['metadata']['exclude']
+
+    for file_location in sorted(files):
+        match = fnmatch.fnmatch(file_location, excluded)
+        if match:
+            log_callback = logcollector.callback_excluded_file(file_location,
+                                                               prefix=prefix)
+            wazuh_log_monitor.start(timeout=60, callback=log_callback, error_message=f"The expected 'File excluded: "

```

Comment:
Please avoid this kind of hard-coded values (`timeout=60`). There are some logcollector class attributes with common timeouts that you could use. Also, Please consider if 60 is a required timeout or it could be decreased to 30 seconds, for example.

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/0745ed3a26b428b592929695d4e10c00605c5dd4